blogs/tests/test_templatetags.py

```diff
@@ -33,9 +29,6 @@ def test_get_latest_entries(self):
             entries[0].pub_date.isoformat(),
             '2013-03-04T15:00:00+00:00'
         )
-        b = Box.objects.get(label='supernav-python-blog')
-        rendered_box = _render_blog_supernav(BlogEntry.objects.latest())
-        self.assertEqual(b.content.raw, rendered_box)
```

Comment:
I removed this because it didn't test much and relied on a side effect of running the `update_blogs` and management command and the previously hardcoded primary key in the `update_blog_supernav` function. All while claiming to be a test for a template tag.

If updating the supernav is still required I can add proper tests.

Reply:
I think having more tests wouldn't hurt :) But it can be done in a separate PR.